### ISSUE TITLE
GIMP uses the default language instead of changed one

### ISSUE THREAD

**Issue by freedom4JA** [netblue30/firejail#4825]
I use GIMP-2.10 and after a time my system-notification asked me to download the german language ( I think its something with gnome in the name)
So far so good. 

When I start GIMP without firejail from now on it uses the german language for GIMP. 
But when I use it with firejail it uses the preset "system language" in english. 
I searched for the problem for a while but couldn't find how to solve. Maybe it don't finds the changed language pack which I installed. Does someone know where to find it respectively to solve the problem. I would like to use german language.

**Comment by glitsj16** [comment]
If I'm not mistaken GIMP installations usually put the language stuff in `/usr/share/locale` (at least that's what I can see on my Arch Linux box). And we **whitelist** that path in [whitelist-usr-share-common.inc](https://github.com/netblue30/firejail/blob/master/etc/inc/whitelist-usr-share-common.inc#L40). What OS and firejail version are you seeing this on?

**Comment by freedom4JA** [comment]
Yes, I mean the programm itself with it's content like "save as", "open new file" ... and "brush" , what I want in german so the **locale** I guess. 
I use Linut Mint 20.2 and the latest firejail 0.9.66
